doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -2,6 +2,7 @@ server_names_hash_bucket_size 64;
 
 server {
     listen 80;
+    listen [::]:80;
```

Comment:
At least for me locally, this configuration does not work... the `/etc/nginx/sites-enabled/default` contains
```
server {
        listen   [::]:80 ipv6only=off default_server;
```
and all other vhosts have to match, so I have to do
```
server {
    listen [::]:80;
    server_name meet.hacksaar.de;
    return 301 https://$host$request_uri;
}
```
for Jitsi to work properly.

Reply:
Ah never mind, looks like that's not actually the default but I forgot I changed that. Sorry.

I can confirm that the configuration you are suggesting here works.